glossary.html

```diff
@@ -55,7 +54,7 @@ <h1> Acronym Glossary </h1>
 <footer> 
     <hr>
  <blockquote>"The successeful Warrior is the average man with laser focus."<br><span class ="byline"><i> &mdash; Bruce Lee </i></span></blockquote>
-<p id="page_copyright">&copy; <a href="mailto:adam@email.com?subject=Message%20from%20CSD120%20Website">Adam Griffith</a></p>
+<p id="page-copyright">&copy; <a href="mailto:adam@email.com?subject=Message%20from%20CSD120%20Website">Adam Griffith</a></p>
 
```

Comment:
I just renamed your id here to bring it in line with what the lab asked for, but what you have works too